src/screens/SignedChatScreen/index.tsx

```diff
@@ -52,6 +52,10 @@ const SignedChatScreen = () => {
     goToChatInfoScreen({from: SIGNED});
   };
 
+  const scrollToEnd = () => {
+    flatlistRef.current?.scrollToEnd();
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `scrollToEnd` has been added. This seems to be a utility function for scrolling to the end of the chat. Ensure that `flatlistRef` is correctly defined and refers to the FlatList component you want to scroll.
```diff
+  const scrollToEnd = () => {
+    flatlistRef.current?.scrollToEnd();
+  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->